src/gui/torrentmodel.h

```diff
@@ -33,15 +34,20 @@
 #define TORRENTMODEL_H
 
 #include <QAbstractListModel>
+#include <QColor>
+#include <QIcon>
+#include <QPixmap>
 #include <QList>
 
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
You are adding a bunch of libtorrent headers for the sake of what? Those static functions below do not need access to the private members of the class. Then why?